src/CSharp/MetadataWebApi/MetadataWebApi.Tests/IMetadataApiExtensionsTests.cs

```diff
@@ -25,29 +24,5 @@ public static class IMetadataApiExtensionsTests
             // Act and Assert
             await Assert.ThrowsAsync<ArgumentNullException>("value", () => value.GetDownloadUriAsync(fileName, fileHash));
         }
-
```

Comment:
@johndarv I was a little torn... there are quite a few tests and behaviours of questionable value, but this one at least was compatible with .Net Core. In any case I'm trying to change the minimum here but feel free to submit your own PR afterwards! :)